Structure partially changed. Creators block added.

valeryvpetrov-dev opened the pull request:
@RuslanMusin , get it reviewed, please.

Commit:
Structure partially changed. Creators block added.

RuslanMusin commented on app/src/main/AndroidManifest.xml:
```diff
@@ -27,17 +29,31 @@
             android:name=".ui.characters.CharactersActivity"
             android:label="@string/characters"
-            android:parentActivityName=".ui.characters_list.CharactersListActivity">
+            android:parentActivityName=".ui.characterslist.CharactersListActivity">
```
Считаю все-таки более удобным для чтения разделение с нижним подчеркиванием. Приятнее глазу просто, ну да ладно, пусть все будет по стандарту

valeryvpetrov-dev replied:
Надо будет узнать мнение Наиля.

RuslanMusin commented on app/src/main/java/com/itis/android/lessontwo/model/creators/Creator.java:
```diff
@@ -0,0 +1,129 @@
+public class Creator implements ListItem {
+
+    @SerializedName("id")
+    @Expose
```
Думаю ты создал эти классы через какой-нить конвертер,так вот тут лишним является одновременное использование Expose и SerializedName. Думаю стоит удалить, так как просто лишнее, хотя по сути не мешает
Вот ссылка https://stackoverflow.com/questions/34752200/gson-expose-vs-serializedname

valeryvpetrov-dev replied:
Этого я не знал, спасибо)

RuslanMusin commented on app/src/main/java/com/itis/android/lessontwo/ui/base/BaseContract.java:
```diff
@@ -0,0 +1,18 @@
+ * Created by valera071998@gamil.com on 02.03.2018.
+ */
+
+public interface BaseContract {
```
Годный интерфейс, молодец , что абстрагировал)

RuslanMusin commented on app/src/main/java/com/itis/android/lessontwo/ui/comics/ComicsActivity.java:
```diff
@@ -50,12 +51,12 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         long id = getIntent().getLongExtra(ID_KEY, 0);
-        new ComicsPresenter(this);
-        presenter.loadComics(id);
+        new ComicsPresenter(this);  // странный способ связывания view и presenter
```
Согласен, немного неочевидный

RuslanMusin commented on app/src/main/res/values/strings.xml:
```diff
@@ -7,14 +7,21 @@
     <string name="drawer_close">Close navigation drawer</string>
 
+    <!--General screen-->
+    <string name="title_name">Name</string>
```
Грамотная обработка общих слов в целом, молодец)

Commit:
Update Creator.java. 

@Expose annotation converts JSON attribute by POJO attribute name.
@SerializedName converts JSON attribute by particular name.

Commit:
Update CreatorsResponse.java

Reason was described in <Update CreatorsResponse.java> commit.

Commit:
Update CreatorsResponseData.java

Reason was described in <Update Creator.java> commit.
